pkg/sidecars/models/factory.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"github.com/inspr/inspr/pkg/meta"
+	"github.com/inspr/inspr/pkg/operator/k8s"
+)
+
+// SidecarConnections object to store a sidecar's connectio ports
+type SidecarConnections struct {
+	InPort  int32
+	OutPort int32
+}
```

Comment:
shouldn't this have json tags

Reply:
This is from another branch actually, I didn't do it